src/extractwindow.cpp

```diff
@@ -4,7 +4,7 @@
 #include <QFileDialog>
 
 ExtractWindow::ExtractWindow(QWidget *parent) :
-    QMainWindow(parent),
+    QDialog(parent),
```

Comment:
I don't know why changes to dialog. I'm need more research about this.

Reply:
Explained on the other CompressWIndow diff